Atomic/rpm_host_install.py

```diff
@@ -33,12 +38,31 @@ def _do_rename_path(path, rename_files):
         return path
 
     @staticmethod
-    def rm_add_files_to_host(old_installed_files, exports, prefix="/", files_template=None, values=None, rename_files=None):
+    def file_checksum(path, blocksize=(1<<20)):
+        if not os.path.exists(path) or os.path.isdir(path):
+            return "0"
+
+        h = hashlib.sha256()
+        with open(path, "r") as f:
+            while True:
+                chunk = f.read(blocksize)
+                if chunk == None or len(chunk) == 0:
+                    break
+                h.update(chunk.encode())
```

Comment:
Q: why do we have an encode here? would it make a difference if we update with chunk directly?

Reply:
good point, it is probably not needed.  But I spotted anothe issue, the file must be opened with 'rb'.  I will fix it